scripts/setup_irods.py

```diff
@@ -106,17 +106,9 @@ def setup_server(irods_config, json_configuration_file=None, test_mode=False):
     if json_configuration_dict is not None:
         # role
         irods_config.commit(json_configuration_dict['server_config'], irods.paths.server_config_path())
-        # database
-        if irods_config.is_provider:
-            from irods import database_interface
-            if database_interface.database_already_in_use_by_irods(irods_config):
-                raise IrodsError('Database specified already in use by iRODS.')
```

Comment:
Removed because the logic that is responsible for creating tables and other things detects the existence of iRODS tables and gracefully skips touching the database instead of throwing an exception.

The idea is that this should also bring both installation paths closer in behavior.

Reply:
I think this is correct.